velox/exec/tests/CMakeLists.txt

```diff
@@ -83,7 +83,8 @@ add_executable(
   ValuesTest.cpp
   WindowFunctionRegistryTest.cpp
   WindowTest.cpp
-  SortBufferTest.cpp)
+  SortBufferTest.cpp
+  PrefixSortTest.cpp)
```

Comment:
nit: Should we keep this list in lexicographic order?

Reply:
That would be nice.